symfony/translation/3.3/etc/packages/translation.yaml

```diff
@@ -0,0 +1,11 @@
+parameters:
+    # Change this locale depending on your app's default language
+    locale: 'en'
```

Comment:
This should be removed, no parameters should be defined under `etc/`. Instead,  you should use the `container` configurator to set this automatically in `container.yaml`